.gitignore

```diff
@@ -73,3 +73,6 @@ sw/host/**/target
 
 # Bazel-related cache and output directories
 bazel-*
+
+# Local binaries that should not be committed to the codebase
+third_party/bin
```

Comment:
Could this be a dot-file at the top of the repo, instead...?

Reply:
Done